README.md

```diff
@@ -156,6 +156,21 @@ SET CI=true
 npm test
 ```
 
+The value for `DASHDB_SCHEMA` must be of the pattern `SCHEMA{build number}_{build name}_{platform}_{node version}`.
+For example : **SCHEMA1_DASHDB_DARWIN_10**.
+
+Alternatively, let `DASHDB_SCHEMA` be computed for you by setting these values instead:
+
+```bash
+export PACKAGE_NAME=loopback-connector-dashdb 
+export BUILD_NUMBER={build number}   For example: 1
+export nodeVersion={node version}    For example: 10
+```
+
+This will create a schema with the name: **SCHEMA1_DASHDB_DARWIN_10**
+
+This pattern of the schema name is important for database **cleanup** and **seeding** purposes.
```

Comment:
+1 to update the README to describe all necessary steps.

I am confused though - why is it necessary to provide all of these properties manually? Why cannot our test suite infer them automatically?

- Read `PACKAGE_NAME` from `package.json`'s property `name`
- Set `BUILD_NUMBER` to arbitrary value (e.g. 1`)
- Set `nodeVersion` to the value returned by `process.versions.node.split('.')[0]` (JavaScript code to be executed in Node.js land)

Reply:
I think this is great for future enhancement. IIRC the pretest was originally created to set these values up using env variables set by Jenkins. @bajtos @emonddr do we want to create a follow up issue for it?

Reply:
@bajtos @b-admike @dhmlau , I opened an issue for this future enhancement : https://github.com/strongloop/loopback-connector-dashdb/issues/83